SR2023/2023-02-23-virtual-competition-reminder.md

```diff
@@ -0,0 +1,29 @@
+---
+to: Student Robotics 2023
+subject: SR2023 Virtual Competition and Tech Day Reminder
+---
+
+Hi
+
+We're really looking forward to see your robots in the [virtual competition](https://studentrobotics.org/events/sr2023/virtual-competition/) this weekend.
+
+[Submit your robot code](https://studentrobotics.org/code-submitter/).
+
+If you're having any issues logging in to the code submitter please let us know as soon as possible. You can log in to the code submitter using your TLA and the same password as you use for Discord.
+
+You have until 8pm (UK time) Friday 24th February to submit your team's latest code. You can upload early, but at that time we'll capture all the uploads. This is your first 4 appearances in the competition, scores you real points, and is a great way to see how you robot's strategy pays off. This will not affect your ability to compete in the [final competition](https://studentrobotics.org/events/sr2023/competition/).
```

Comment:
```suggestion
You have until 8pm (UK time) Friday 24th February to submit your team's latest code. You can upload early, but at that time we'll capture all the uploads. These matches are your first 4 appearances in the competition, scoring you valuable league points. It's also a great way to see how you robot's strategy fares against other robots. Not entering will not affect your ability to compete in the [final competition](https://studentrobotics.org/events/sr2023/competition/) but will put you at a disadvantage to the teams that have entered our virtual event.
```

Reply:
Comment from Peter, we could drop the last sentence entirely. I think I agree but up to you @RealOrangeOne